Gopkg.toml

```diff
@@ -59,11 +59,11 @@ required = [
 
 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
-  branch = "master"
+  branch = "0.1.0"
```

Comment:
@jichenjc this works

```
diff --git a/Gopkg.toml b/Gopkg.toml
index b1a834c..af2b525 100644
--- a/Gopkg.toml
+++ b/Gopkg.toml
@@ -59,7 +59,7 @@ required = [

 [[constraint]]
   name = "sigs.k8s.io/cluster-api"
-  branch = "master"
+  branch = "release-0.1"

 [[constraint]]
   name = "sigs.k8s.io/controller-runtime"
```

Reply:
@jichenjc please check aws, it was using `release-0.1` as well.

Reply:
I already changed this locally to 'version'... weird